src/xsl/openrosa2html5form.xsl

```diff
@@ -265,7 +254,9 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
                 <xsl:if test="$binding/@relevant">
                     <xsl:value-of select="'or-branch pre-init '"/>
                 </xsl:if>
-                <xsl:call-template name="appearance" />
+                <xsl:if test="@appearance">
+                    <xsl:value-of select="'or-appearances-placeholder '"></xsl:value-of>
+                </xsl:if>
```

Comment:
When I initially moved the appearances logic into `transform.ts`, I was appending the classes to the end of each element's `classList`. The `or-appearances-placeholder` class is used in cases where appearance classes were _not_ expected to be appended to the end. This is done because unlike attribute order (which should not matter), the ordering of classes may affect CSS precedence.

This potential regression was caught by Enketo Core's tests.